modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -1,158 +1,110 @@
-+
+
+Restart Taskomatic:
 ----
-spacewalk-common-channels -u admin -p `secret` -a x86_64 'centos7'
-spacewalk-common-channels -u admin -p `secret` -a x86_64 'centos7-updates'
-spacewalk-common-channels -u admin -p `secret` -a x86_64 'centos7-uyuni-client-x86_64'
+systemctl restart taskomatic
 ----
 
-.Procedure: Synchronizing {centos} Clients
-
-. In the {productname} {webui}, navigate to menu:Main Menu[Software > Manage], and select the base channel you want to synchronize.
-. In the [guimenu]``Repositories`` tab, navigate to the [guimenu]``Sync`` subtab, and click btn:[Sync Now].
-You can also create a regular synchronization schedule on this page.
-
-When you have prepared you {productname} Server, you can install your {centos} client using your preferred installation media and method.
+== {centos} Channel and Repository Management
 
-.Procedure: Setting up a {centos} Client
+.Procedure: Add the Channels and Repositories
 
-. At the command prompt, copy all relevant GPG keys to [path]``/srv/www/htdocs/pub``.
-If you intend to use a bootstrap script to register your client, you can add the GPG keys to your bootstrap script.
-. Check that the client machine can resolve itself and your {productname} Server using DNS.
-. Check that there is an entry in [path]``/etc/hosts`` for the real IP address of the client.
-. Create an activation key called `centos7` on the {productname} Server that points to the correct parent and child channels, including the {centos} base repository, updates, and client channels.
+ifeval::[{suma-content} == true]
+SUSE Manager subscriptions entitle everyone to the tools channels for {sleses} (also known as Red Hat Expanded Support or RES). Any Red Hat Enterprise Linux system should use these to create the proper bootstrap repository for either traditional or salt-minion connectivity.
 
-When you are ready to register your {centos} client, follow the instructions in xref:client-configuration:registration-overview.adoc[].
+1. Add the corresponding required {slesesa} channels and allow it to synchronize from SUSE Customer Center.
 
+{slesesa} 6::
+* "RHEL6 Base x86_64" and "SUSE Linux Enterprise Client Tools RES6 x86_64" at the WebUI, or [systemitem]``rhel-x86_64-server-6`` and [systemitem]``res6-suse-manager-tools-x86_64`` at CLI.
 
-////
-This is all duplicated content. LKB 2018-08-31
+{slesesa} 7::
+* "RHEL7 Base x86_64" and "SUSE Linux Enterprise Client Tools RES7 x86_64" at the WebUI, or [systemitem]``rhel-x86_64-server-7`` and [systemitem]``res7-suse-manager-tools-x86_64`` at CLI.
 
-Now prepare the bootstrap script.
+You can use [command]``mgr-sync`` to do it from CLI.
 
-[[proc.bp.expanded-support.centos-repos.trad.bsscript]]
-.Procedure: Preparing the Bootstrap Script
-. Create/edit your bootstrap script to correctly reflect the following:
-+
+// I am not sure we need this, as we instruct users to sync the repositories, including "os" (pool) and updates. I asked clarification from Donald.
```

Comment:
@dvosburg what about this comment? Are you sure users need the ISO?

IIRC when I tested CentOS7 with Uyuni (using Uyuni client tools, but should be the same), I only synced OS and UPDATES CentOS repositories, and that was enough, no need to import the ISO or the DVD.

And I think the same happens for pure RHEL clients when using CDN/Satellite/whatever. As long as OS and Updates repositories are present, the user should be fine.

Or am I missing something?

Reply:
No - the ISO is not necessary.  The challenge is in creating the bootstrap repository for the product.  Are we planning on using 'mgr-create-bootstrap-repo --with-custom-channels'?

If so, then it is important to have the CentOS/RHEL channels cloned into the zero-package base channel the product expects.

Reply:
> Are we planning on using 'mgr-create-bootstrap-repo --with-custom-channels'?

Yes. That was already part of your guide for "pure" RHEL clients and I migreated to the `clients-rh.adoc` here.

> If so, then it is important to have the CentOS/RHEL channels cloned into the zero-package base channel the product expects.

I guess you mean the "CentOS/RHEL" channels need to be child channels of the zero-package base channel, as in your guide for RHEL CDN, right?

I will make sure we have it this way at CentOS as well (for RHEL it's the case already, since I migrated your guide)